src/editor/components/EditorElement.tsx

```diff
@@ -66,19 +67,23 @@ export const EditorElement: React.FC<EditorElementProps> = ({
       isDragging: !!monitor.isDragging(),
     }),
   });
-  const classes = useEditorElementStyles({ isDragging });
+  const [selection, setSelection] = useSelection();
+  const classes = useEditorElementStyles({
+    isDragging,
+    isSelected: selection === wrappedElement,
+  });
   const uiPath = getUISchemaPath(wrappedElement);
   const dispatch = useDispatch();
 
   return (
-    <Grid
-      item
-      container
-      wrap='nowrap'
-      direction='column'
+    <div
       data-cy={`editorElement-${uiPath}`}
       className={classes.editorElement}
```

Comment:
wouldn't it be easier to pass two classnames depending on the selection instead of changing the styling all the time?

Reply:
I don't think there's a difference in performance between the two approaches, but I switched to using classnames because it made the styles a bit cleaner.